Makefile

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY: all csi-snapshotter clean test
+.PHONY: all csi-snapshotter-common csi-snapshotter-sidecar clean test
```

Comment:
Prefer calling the binary "controller" instead of "common"

Reply:
changed to "snapshot-controller" for the common controller and "csi-snapshotter" for the sidecar.